test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -1133,11 +1133,11 @@
                     "lib/netstandard1.6/Microsoft.Azure.DocumentDB.Core.dll": {}
                 }
             },
-            "Microsoft.Azure.Functions.JavaWorker/1.8.0": {},
-            "Microsoft.Azure.Functions.NodeJsWorker/2.0.6": {},
-            "Microsoft.Azure.Functions.PowerShellWorker.PS6/3.0.557": {},
-            "Microsoft.Azure.Functions.PowerShellWorker.PS7/3.0.560": {},
-            "Microsoft.Azure.Functions.PythonWorker/3.1.1.8": {},
+            "Microsoft.Azure.Functions.JavaWorker/1.8.1": {},
+            "Microsoft.Azure.Functions.NodeJsWorker/2.1.0": {},
+            "Microsoft.Azure.Functions.PowerShellWorker.PS6/3.0.630": {},
+            "Microsoft.Azure.Functions.PowerShellWorker.PS7/3.0.629": {},
+            "Microsoft.Azure.Functions.PythonWorker/3.1.1.10": {},
```

Comment:
All of these worker and WebJobs changes are fine. We skip this during our diff so they just get updated like this whenever someone checks in a new file.